hack/operatorhub/publish-bundle.sh

```diff
@@ -96,6 +100,9 @@ BUNDLE_DIR="${CUR_DIR}"/../../bundle
 mkdir -p "${OPERATORS_DIR}"
 cp -a "${BUNDLE_DIR}"/. "${OPERATORS_DIR}"
 
+skipInDryRun git config user.name "${GIT_CONFIG_USER_NAME}"
```

Comment:
I think this should be wrapped in an if-clause that checks whether `git config user.name` returns a value - if it's already set, no need to overwrite